cockpit/image-builder-frontend.spec

```diff
@@ -0,0 +1,37 @@
+Name:           cockpit-composer
+Version:        0
```

Comment:
Should be version 53, that's the version cockpit-composer is at. Once we release it should be 54. Could just start on 54 to be on the safe side.